cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1382,6 +1383,31 @@ def _check_spatial_obs(self):
         self._validate_spatial_tissue_position("array_row", 0, 77)
         self._validate_spatial_tissue_position("in_tissue", 0, 1)
 
+        # Validate cell type.
+        self._validate_spatial_cell_type_ontology_term_id()
+
+    def _validate_spatial_cell_type_ontology_term_id(self):
+        """
+        Validate cell type ontology term id is "unknown" if Visium, is_single is True and in_tissue is 0.
+
+        :rtype none
+        """
+        # Exit if:
+        # - not Visium and is_single is True as no further checks are necessary
+        # - in_tissue is not specified as checks are dependent on this value
+        if not self._is_visium_and_is_single_true() or "in_tissue" not in self.adata.obs:
+            return
+
+        # Validate cell type: must be "unknown" if Visium and is_single is True and in_tissue is 0.
+        if (
+            (self.adata.obs["assay_ontology_term_id"] == ASSAY_VISIUM)
+            & (self.adata.obs["in_tissue"] == 0)
+            & (self.adata.obs["cell_type_ontology_term_id"] != "unknown")
+        ).any():
+            self.errors.append(
+                f"obs['cell_term_ontology_term_id'] must be 'unknown' when {ERROR_SUFFIX_VISIUM_AND_IS_SINGLE_TRUE_IN_TISSUE_0}."
```

Comment:
Slightly different minor typo of 'cell_term_ontology_term_id', should be 'cell_type_ontology_term_id'.

Otherwise, the logic looks good!